src/components/ProtectedRoute/ProtectedRoute.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+export default function ProtectedRoute({ children, authorized, ...props }) {
+  return <Route {...props}>{authorized ? children : null}</Route>;
```

Comment:
@uriklar maybe use `<Redirect path="sign-up">` instead of null?

Reply:
Done